modules/java_api/src/test/java/org/intel/openvino/README.md

```diff
@@ -0,0 +1,42 @@
+## Build
```

Comment:
Why not keep README at `modules/java_api/src/test`?

Reply:
Moved. Thank you